.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,214 @@
+name: PHPUnit
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+env:
+  APP_NAME: guests
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+
+    strategy:
+      # do not stop on another job's failure
+      fail-fast: false
+      matrix:
+        php-versions: ['7.4']
+        databases: ['sqlite']
+        server-versions: ['stable18']
```

Comment:
I assume it should actually be tested against master?

Reply:
it wasn't previously, so I put back 18 :shrug: 
if we want master we just need to enable the app with `--force`, but we can also do that :)